posixlib/src/main/scala/scala/scalanative/posix/unistd.scala

```diff
@@ -33,6 +33,10 @@ object unistd {
   def truncate(path: CString, length: off_t): CInt = extern
   def unlink(path: CString): CInt = extern
   def usleep(usecs: CUnsignedInt): CInt = extern
+  def nanosleep(
+      requested: Ptr[time.timespec],
+      remaining: Ptr[time.timespec]
+  ): CInt = extern
```

Comment:
This should be in the time header as far as I can tell. https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/time.h.html

Reply:
Thank you for catching this up, I don't know why I've put it in unistd at this point.